test/common/access_log/access_log_impl_test.cc

```diff
@@ -3,6 +3,7 @@
 #include <memory>
 #include <string>
 
+#include "envoy/config/core/v3alpha/health_check.pb.validate.h"
```

Comment:
Instead of including this everywhere, include it where the utility function is actually defined/used.